README.md

```diff
@@ -30,16 +38,27 @@ Next, use this command to create the databases:
 rake db:create
 ```
 
-<!-- * Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.) -->
-
-## Deployment instructions
+## Usage
 
 Finally, use the following command to initialize the live server:
 
 ```
 rails s
 ```
+
+## Author
+
+👤 David Vergaray
+
+- GitHub:   [@Yothu](https://github.com/Yothu)
+- Twiter:   [@Daivhy](https://twitter.com/Daivhy)
+- LinkedIn: [David Vergaray](https://www.linkedin.com/in/david-vergaray-almontes-051a11127/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] Kindly note that you still need to add the `Acknowledgments` in your readme as requested by the previous reviewer. [Microverse Readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)